public/test.js

```
@@ -0,0 +1 @@
+console.log(1)
```

ESLint

[error] Insert `⏎`

(prettier/prettier)